.github/workflows/ci.yml

```
@@ -14,85 +14,85 @@ jobs:
     strategy:
       matrix:
         destination: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)